rs/replicated_state/src/canister_state.rs

```diff
@@ -325,18 +325,11 @@ impl CanisterState {
     }
 
     /// Inducts messages from the output queue to `self` into the input queue
-    /// from `self` while respecting queue capacity and subnet available memory.
+    /// from `self` while respecting queue capacity and subnet's available
+    /// guaranteed response memory.
```

Comment:
There are some 100 occurrences of `subnet_available_memory` throughout `ic_replicated_state`. I figured that it's enough to update the doc comments to make it clear that this is the "guaranteed response message memory" instead of renaming all occurrences to an excessively long name. Also, there will never be an "available best-effort message memory" argument passed around or field, so I felt it kind of unnecessary to spell it out.

But I can definitely revisit that decision if you think it is worth it.

Reply:
It's fine, I just wasn't entirely sure from reading the comment if this referred to the variable or something else. Left a suggestion.